css/style.css

```diff
@@ -17,3 +17,325 @@ body {
  */
 
 
+ body {
+  background-color: black;
+ }
+   
+ h1,h2,h3,h4,h5,h6 {
+   margin-top: 0;
+   margin-bottom: 0;
+ }
+ 
+ /*
+ * Remove default margin.
+ */
+
+ p {
+  margin-top: 0;
+  margin-bottom: 0;
+  }
+
+
+/* Header */
+
+.header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    height: 5rem;
+    margin-bottom: 1rem;
+    padding-top: 1rem;
+  }
+  
+ 
+  .karma-logo {
+    width: 2rem; /* if you set NO width, the image will not show! */
+    /* add a border around the logo */
+    border: 0px solid white;
+    padding:0rem;
+    padding-top: 20px;
+    object-fit: scale-down;  
+
+  } 
+  
+  /* Navigation */
+  
+  .navigation-list {
+    list-style-type: none;
+    /* remove browser default settings */
+    margin: 0;
+    padding: 0;
+    
+
+    overflow: hidden;
+    background-color: white;
+  }
+  
+  /* determine how each item of the naviagtion bar is shown */
+
+  .navigation-item {
+    color: #fff;
+    padding: 22px 16px 16px 22px;
+    text-decoration: none;
+    display: inline;
+    float: left;
+    font-size: 22px;
+  }
+
+  /* determine how each link is shown */
+
+   .navigation-link {
+    color: gray;
+    display: block;
+    text-align: center;
+    padding: 14px 0px;
+    text-decoration: none;    
+      }
+
+
+
+  /* change colour when mouse hovers over it */    
+
+  .navigation-link:hover {
+        color:#ec5800; /* persimmon */
+      }
+
+     /* always show 'Meet Karma' in black */
+     #meetKarma {
+      color: black;
+      font-weight:500;
+   }  
+
+
+   /* FLEXBOX for the LOGO & Navigation Bar */
+
+   .LogoNavbox{
+    display: flex;
+    justify-content: space-between;
+    background-color: white;
+    margin-left: 7.3rem;
+    margin-right: 7.3rem;
+    padding-left: 5rem;  /* MOVE TO THE RIGHT */
+    padding-right: 5rem; /* MOVE TO THE LEFT */
+    
+  }
+
+  /* The main image i.e. the 'hero' */ 
+
+.mainImage {
+    background-image: url(../img/first-background.jpg);
+    background-repeat: no-repeat;
+    background-position: center;
+ 
+    /* THE ACTUAL SIZE OF THE IMAGE IS 1600 * 735 PIXELS */
+    height: 735px; /* if you set NO height, the image will not show! */
+    width: 100%;
+    /* Handle how the text is displayed on top */
+    object-fit:cover;
+    position: relative;
+    text-align: top left;
+    color: white;
+
+  }
+
+  /* Based on https://www.w3schools.com/howto/howto_css_image_text.asp */
+
+  .centred {
+    position: absolute;
+    top: 36%;
+    left: 46%;
+    transform: translate(-50%, -50%);
+  }
+
+  .para1 {
```

Comment:
What is this class name telling us? That it's the first paragraph? That sounds more like an id than a class name, which we should avoid if possible. When naming and separating your classes think about how to make them as reusable as possible. If you have to specify something only really occurs in one specific scenario, see if you can specify that with a selector that uses semantic elements or parent elements (e.g. `.mainImage > h2` or something.)